lib/callbacks.js

```diff
@@ -44,8 +44,7 @@ const once = fn => {
 // It's unsafe: may return null, allows multiple calls
 //   args - array, arguments
 // Returns: function, callback or null
-// Hint: previous name: `common.cbUnsafe` (deprecated)
-// Hint: another alias: `common.extractCallback` (deprecated)
+// Deprecated: previous names: `common.cbUnsafe`, `common.extractCallback`
```

Comment:
Is `unsafeCallback` deprecated? Because this comment tells that it is.
I think that this comment meant that the previous name (common.cbUnsafe) is deprecated, not this function.

Reply:
That's why is asked it here in https://github.com/metarhia/common/pull/194#discussion_r219679795. We need special label to store previous names. Other options are to remove them at all or copy-paste code from the time when they were present, but I'd like not to have something like this

Reply:
We don't care about previous names, that's not how deprecation works. You deprecate the *previous* function with smth like `Deprecated: use newName instead` and the `newName` don't have anything about the previous names.

If the `common.cbUnsafe`, `common.extractCallback` are already removed then just remove this comment as it is not needed.